### DISCUSSION TITLE
Can you do a 2-way-highlight?

### DISCUSSION THREAD

**Discussion by hoangthisd** [yandeu/five-server#71] [Help / Questions]
When I click on an HTML tag in Visual Studio Code, the live preview's respective components will be highlighted, that's very useful. 
Can I do the opposite? When I click on something on the live preview web browser, the text cursor will jump to the corresponding code in the editor. This was what Brackets editor used to do.

**Comment by candideu** [comment]
This would be incredible, and would really help beginner learners.

**Comment by simonferndriger** [comment]
DreamWeaver had this functionality already decades ago. It is super useful, because you can visually click on the part of the website you want to edit and jump right to it in the code. You can save a huge amount of time this way.

**Comment by withkynam** [comment]
This would be extremely useful for me. Does anyone knows when this will come out?
